styles.css

```diff
@@ -18,9 +18,12 @@ aside{
   flex: 1;
   background:#ddd;
 }
-.main{
+main{
   flex: 3;
 }
+header>h1, footer>h3, footer>p, main>h2, main>div, aside>h2 {
+  margin: 0;
+}
```

Comment:
this is a good way to do this. could have also done something like `header > * ` to select all direct children. Another thing to do would have been assigning classes `h1 = "heading"`.